n64.mk

```diff
@@ -7,23 +7,26 @@ N64_ROM_SAVETYPE = # Supported savetypes: none eeprom4k eeprom16 sram256k sram76
 N64_ROM_RTC = # Set to true to enable the Joybus Real-Time Clock
 N64_ROM_REGIONFREE = # Set to true to allow booting on any console region
 
+# Override this to use a toolchain installed separately from libdragon
```

Comment:
How will this help the later changes from unstable?

Reply:
It doesn't actually, it's just to help me to avoid conflicts when merging n64.mk.

Reply:
I don't know how we use it but I can check for myself

Reply:
The usecase is coming from the Arch Linux mips64 package which is trying to be a "generic mips64 compiler" that can be used also for romhacks or libultra work, or even non-N64 work. So it is installed under `/usr` like a regular package. Then on top of that there is a different "libdragon" package which gets installed to `/opt/libdragon`.